mixins/property-required/test/property-required-mixin.test.js

```diff
@@ -81,88 +73,70 @@ describe('PropertyRequiredMixin', () => {
 		const tag = defineCE(
 			class extends PropertyRequiredMixin(LitElement) {
 				static properties = {
-					requiredAttr: { attribute: 'required-attr', type: String }
+					requiredAttr: { attribute: 'required-attr', type: String, required: true }
 				};
-				constructor() {
-					super();
-					this.addRequiredProperty('requiredAttr');
-				}
 			}
 		);
 		const elem = await fixture(`<${tag}></${tag}>`);
 		expect(() => elem.flushRequiredPropertyErrors())
 			.to.throw(TypeError, createDefaultMessage(tag, 'required-attr'));
 	});
 
-	it('should throw if no property is defined', async() => {
-		const elem = await fixture(`<${tagValid}></${tagValid}>`);
-		expect(() => elem.addRequiredProperty('missing'))
-			.to.throw(createUndefinedPropertyMessage(tagValid, 'missing'));
-	});
-
 	it('should throw if type is non-String', async() => {
 		const tag = defineCE(
 			class extends PropertyRequiredMixin(LitElement) {
 				static properties = {
-					attr: { type: Boolean }
+					attr: { type: Boolean, required: true }
 				};
 			}
 		);
-		const elem = await fixture(`<${tag}></${tag}>`);
-		expect(() => elem.addRequiredProperty('attr'))
+		const elem = await fixture(`<${tag} attr="value"></${tag}>`);
+		expect(() => elem.flushRequiredPropertyErrors())
 			.to.throw(createInvalidPropertyTypeMessage(tag, 'attr'));
 	});
 
-	it('should use custom message', async() => {
+	it('should not throw if type is undefined (defaults to String)', async() => {
```

Comment:
Added a test case for if they don't include a `type`, which the Lit docs say defaults to `String`.